MekHQ/src/mekhq/campaign/unit/Unit.java

```diff
@@ -3726,8 +3742,19 @@ public boolean isGunner(Person person) {
         return false;
     }
 
+    /**
+     * Checks whether a person is considered the commander of this unit.
+     * 
+     * @param person A <code>Person</code> in the campaign. The person need not be assigned to the unit as
+     *               crew, in which case the return value will be false.
+     * @return       Whether the person is considered the unit commander. A unit with no crew has no
+     *               commander and this method will return false.
+     * 
+     * @see {@link #getCommander()}
+     */
     public boolean isCommander(Person person) {
-        return person.getId().equals(getCommander().getId());
+        Person commander = getCommander();
+        return (null != commander) && person.getId().equals(commander.getId());
```

Comment:
Probably should add a null-check for "person" just to be on the safe side.

Reply:
Agreed